### ISSUE TITLE
Obscure claim in the doc: hyperbolic triangle

### ISSUE THREAD

**Issue by stla** [CGAL/cgal#6578]
Hello,

In the documentation about hyperbolic Delaunay triangulations, it is claimed that such a triangle is not hyperbolic:

![hdelaunaycgal](https://user-images.githubusercontent.com/4466543/168145707-839abe95-9dd8-4cba-9e0f-904f6c95f51d.png)

However, mathematically speaking, this triangle is not less hyperbolic than another one. This confused me. IMHO it would be better to use another wording, such as "Delaunay-hyperbolic".

**Comment by MoniqueTeillaud** [comment]
Can you please give the precise reference of the place where you find this claim about a _triangle_ ? I cannot find it. 

What I find is "The shaded face is non-hyperbolic." (caption of [Figure 43.2](https://doc.cgal.org/latest/Hyperbolic_triangulation_2/index.html#fig__Hyperbolic_triangulation_2Euclidean_vs_hyperbolic) 

The claim is about a _face_ (an object of the triangulation), not about a triangle (a geometric object). 
This terminology is used consistently in the whole package.

**Comment by stla** [comment]
You're right, it's about faces. But it is also said "simplex", and a simplex is rather a triangle, no? It is also written "the circumcircle (of a face)" but we rather talk about the circumcircle of a triangle. Otherwise you're right, this is consistent.
